drivers/usb/device/usb_dc_native_posix.c

```diff
@@ -427,15 +420,15 @@ int usb_dc_ep_read_wait(uint8_t ep, uint8_t *data, uint32_t max_data_len,
 
 int usb_dc_ep_read_continue(uint8_t ep)
 {
-	uint8_t ep_idx = USBIP_EP_ADDR2IDX(ep);
+	uint8_t ep_idx = USB_EP_GET_IDX(ep);
 
 	if (!usbip_ctrl.attached || !usbip_ep_is_valid(ep)) {
 		LOG_ERR("Not attached / Invalid endpoint: EP 0x%x", ep);
 		return -EINVAL;
 	}
 
 	/* Check if OUT ep */
-	if (USBIP_EP_ADDR2DIR(ep) != USB_EP_DIR_OUT) {
+	if (USB_EP_GET_DIR(ep) != USB_EP_DIR_OUT) {
```

Comment:
s/a

Reply:
No, I will not change this logic here. Also not relevant to this PR.